### DISCUSSION TITLE
dotcover alias and using coverage.xml

### DISCUSSION THREAD

**Discussion by minoseah629** [cake-build/cake#3834] [Q&A]
does the dotcover alias support using coverage.xml as argument? 

https://www.jetbrains.com/help/dotcover/Running_Coverage_Analysis_from_the_Command_LIne.html#basic

**Comment by nils-a** [comment]
[`DotCoverCover`](https://cakebuild.net/api/Cake.Common.Tools.DotCover/DotCoverAliases/) has an overload that takes a [`DotCoverCoverSettings`](https://cakebuild.net/api/Cake.Common.Tools.DotCover.Cover/DotCoverCoverSettings/) settings. In there you will find what you are looking for.
